mlrun/model_monitoring/helpers.py

```diff
@@ -278,9 +278,13 @@ def calculate_inputs_statistics(
     return inputs_statistics
 
 
-def get_endpoint_record(project: str, endpoint_id: str):
+def get_endpoint_record(
+    project: str,
+    endpoint_id: str,
+    secret_provider: typing.Optional[typing.Callable] = None,
```

Comment:
Only if you want to be extra with the type hints - what are the input and output types of this callable?